### DISCUSSION TITLE
[Unstable] Some suits still have "PropulsionSpeedMultiplier"

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#13996] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

From the changelog:
> Added "PropulsionSpeed" StatValue and use it instead of "PropulsionSpeedMultiplier" in status effects.

Most items were edited to use the new StatValue but a few still have "PropulsionSpeedMultiplier"
- combatdivingsuit
`<StatusEffect type="OnWearing" target="Character" ObstructVision="true" PressureProtection="5750.0" SpeedMultiplier="0.95" PropulsionSpeedMultiplier="1.4" LowPassMultiplier="0.2" setvalue="true" disabledeltatime="true">`
- makeshiftarmor
`<StatusEffect type="OnWearing" target="Character" SpeedMultiplier="0.7" PropulsionSpeedMultiplier="0.5" setvalue="true" disabledeltatime="true" />` 
- slipsuit
`<StatusEffect type="OnWearing" PropulsionSpeedMultiplier="1.4" />`

### Reproduction steps

1. Use underwater scooter while wearing a combat diving suit

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.5.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested the changes for this and found no issues.
Closing discussion as resolved.
